sample/SampleAspNetCoreApp/Controllers/ValuesController.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace SampleAspNetCoreApp.Controllers
+{
+	/**
+	 * The values controller contains a post method for testing the 'CaptureBody' config option of the dotnet apm module
+	 */
+	[Route("api/Values")]
+	public class ValuesController : Controller
+	{
+		// GET api/values
+		//[HttpGet]
+		//public async Task<ActionResult<IEnumerable<string>>> Get()
+		//{
+		//	var lastValue = "call to elastic.co: ";
+		//	try
+		//	{
+		//		var httpClient = new HttpClient();
+		//		var elasticRes = await httpClient.GetAsync("https://elastic.co");
+		//		lastValue += elasticRes.StatusCode;
+		//	}
+		//	catch
+		//	{
+		//		lastValue += "failed";
+		//	}
+
+		//	return new[] { "value1", "value2", lastValue };
+		//}
+
+		//// GET api/values/5
+		//[HttpGet("{id}")]
+		//public ActionResult<string> Get(int id) => "value" + id;
+
+		[HttpPost("{id}")]
+		public ActionResult<string> Post([FromForm]string id) => "value=" + id;
```

Comment:
Since this is basically a 1 liner, I think it's fine to add it to the `HomeController` - just add a comment to it that describes the use case - basically what you have above.

Reply:
OK. Done. Values controller has been removed and I added the post method to the Home controller with a custom route in order not to affect any of the functionality.

Reply:
Code committed.